app/src/main/kotlin/cz/covid19cz/erouska/ui/dashboard/DashboardFragment.kt

```diff
@@ -219,6 +235,17 @@ class DashboardFragment : BaseFragment<FragmentPermissionssDisabledBinding, Dash
         navigate(R.id.action_nav_dashboard_to_nav_welcome_fragment)
     }
 
+    private fun isPlayServicesObsolete(): Boolean {
+        val current = PackageInfoCompat.getLongVersionCode(
```

Comment:
necrashne to, když Play Services vůbec nebudou? Pro jistotu bych to obalil do try.

Reply:
Souhlas